src/Components/ServiceScene/LogsListScene.tsx

```diff
@@ -238,6 +160,18 @@ export class LogsListScene extends SceneObjectBase<LogsListSceneState> {
 
   private getVizPanel() {
     this.lineFilterScene = new LineFilterScene();
+
+    // If this is called before the query has executed, we need to return an empty data node to init the loading state, instead of no-data
+    const data =
```

Comment:
I think this can be cleaned up now, this was a workaround to stop the flickering caused by having multiple queries to different endpoints in the same query provider.